app/forms/flood_risk_engine/steps/partnership_form.rb

```diff
@@ -29,7 +29,7 @@ def partner_count
         presence: {
           message: t(".errors.full_name.blank")
         },
-        "flood_risk_engine/text_field_content" => {
+        "flood_risk_engine/organisation_name_content" => {
```

Comment:
I think partnerships are made of people names rather so this should use same validator as Individual (text_field_content)